atomic-openshift-install should only output relevant information

tbielawa opened the pull request:
With respect to https://github.com/openshift/openshift-ansible/issues/2445

tbielawa commented:
aos-ci-test

detiber commented on utils/etc/ansible.cfg:
```diff
@@ -19,10 +19,15 @@ nocows = 1
+# control_path
+
+stdout_callback = openshift_quick_installer
+callback_whitelist = openshift_quick_installer
```
Are these actually needed? I was under the assumption that callbacks only needed to be whitelisted when they are disabled by  default.

detiber commented on utils/etc/ansible.cfg:
```diff
@@ -19,10 +19,15 @@ nocows = 1
 
 retry_files_enabled = False
 
+deprecation_warnings=False
```
Is it not possible to filter these out in the callback plugin? Maybe it's not a big deal for the quick installer, but I know we are getting ready to hit a point where at least some of the deprecation warnings are going to be failures with a newer 2.2 build.

tbielawa commented:
Rebased and squashed. Unit tests are not passing. Failures due to introduction of new `ansible-quiet.cfg` config file.

Still a work in progress. Handlers are not 'handled' yet and other acceptance criteria have not yet been met (such as making sure a log file of the complete run is still generated in regular ansible format)

tbielawa commented:
aos-ci-test

tbielawa commented:
@abutcher @smunilla @detiber  PTAL/Provide Feedbacks

tbielawa commented:
@abutcher @smunilla @detiber can you guy try to review this PR today? Much thanks.

jiajliu commented on utils/src/ooinstall/cli_installer.py:
```diff
@@ -876,13 +870,15 @@ def cli(ctx, unattended, configuration, ansible_playbook_directory, ansible_conf
         # If we're installed by RPM this file should exist and we can use it as our default:
         oo_cfg.settings['ansible_config'] = DEFAULT_ANSIBLE_CONFIG
 
+    if os.path.exists(DEFAULT_ANSIBLE_CONFIG):
```
Does "DEFAULT_ANSIBLE_CONFIG" here should be "QUIET_ANSIBLE_CONFIG"??

jiajliu replied:
Does "DEFAULT_ANSIBLE_CONFIG" should be "QUIET_ANSIBLE_CONFIG" here?

tbielawa replied:
Good catch!

tbielawa commented:
aos-ci-test

tbielawa commented:
Rebased onto latest master.

Fixed issue pointed out by @jiajliu 

Includes a conditional message at the end of the playbook run to let users know why plays may have been skipped. 

* https://gist.github.com/tbielawa/f904a5aab1966c2d7fac045732af5bc0#file-final-suggestion

> Installation Complete: Note: Play count is an estimate and some were skipped because your install does not require them

Also, removed the final 'press a key to continue' prompt.

@smunilla @abutcher @sdodson ready for review/merge after CI finishes.
